src/rules/color-named/README.md

```diff
@@ -137,3 +137,45 @@ a {
   composes: red from './index.css';
 }
 ```
+
+### `ignore: ["inside-function"]`
+
+Ignore colors that are inside a function.
+
+For example, with `"never"`:
+
+The following patterns are *not* considered warnings:
+
+```css
+a {
+  color: map-get($colour, blue);
+}
+```
+
+```css
+a {
+  background-image: url(red);
+}
+```
+
+For example, with `"always-where-possible"`:
```

Comment:
@jeddy3 Does it mean that I should restrict this option in code only to `never`? Or this option should also be available in `always-where-possible`?